$OpenBSD: patch-collects_compiler_embed_rkt,v 1.1 2016/01/13 02:05:21 juanfra Exp $

"raco distribute" only works correctly when "raco exe" generates
files with the write permission enabled.

--- collects/compiler/embed.rkt.orig	Tue Jan 12 03:24:07 2016
+++ collects/compiler/embed.rkt	Tue Jan 12 03:47:11 2016
@@ -1820,4 +1820,4 @@
 ;; Restores old permissions (if not #f)
 (define (done-writable dest-exe old-perms)
   (when old-perms
-    (file-or-directory-permissions dest-exe old-perms)))
+    (file-or-directory-permissions dest-exe #o755)))
